packages/arb-avm-cpp/speedtest/speed_test.go

```diff
@@ -62,6 +63,7 @@ func runExecutableFile(b *testing.B, filePath string) {
 	if err != nil {
 		b.Fatal(err)
 	}
+	defer runtime.KeepAlive(cursor)
```

Comment:
Similar to above, I'm confused why we need these outside of the direct cgo interaction

Reply:
:+1: